docs/howto/wallet.md

```diff
@@ -0,0 +1,232 @@
+---
+sidebar_position: 3
+title: Vega Wallet
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import GitPod from './_gitpod.mdx'
+
+# How to use Vega Wallet APIs
+
+## Introduction
+
+To submit orders and other commands on Vega, users need to sign transactions with a private and public key pair. Users on Vega are pseudonymous and are referenced by their public keys. To assist with API integration there is a **REST API service** served from the Vega Wallet software providing signing and key management. 
+
+The Vega Wallet software can store many wallets, and each wallet can store many key pairs. 
+
+Common actions for a wallet include; **logging in**, **retrieving a key pair** and **signing transactions** (such as an order submission). There are several useful how-to guides below that will help with management of keys and signing.
+
+
+## Logging in to an existing wallet
+
+Connect to a Vega Wallet server and log in. This can be either a hosted wallet, one you run yourself locally or stored in the cloud. The resulting `token` will be used to authenticate with the wallet for other actions (such as signing):
```

Comment:
```suggestion
Start a Vega Wallet server and connect to an existing wallet. This can be either a hosted wallet, or one you run yourself locally or stored in the cloud. The resulting `token` will be used to authenticate with the wallet for other actions (such as signing):
```